engine/access/rpc/backend/backend_events.go

```diff
@@ -45,6 +45,12 @@ func (b *backendEvents) GetEventsForHeightRange(
 		return nil, status.Errorf(codes.Internal, " failed to get events: %v", err)
 	}
 
+	// start height should not be beyond the last sealed height
```

Comment:
if in the `GetEventsForHeight` request, the `start height` is `>` than the finalized height, then return an error (instead of returning no events)